composite/token-allocation/src/config.ts

```diff
@@ -21,24 +15,56 @@ enum DataProvider {
   Kaiko = 'kaiko',
 }
 
-const providers: Record<string, PriceAdapter> = {
-  [DataProvider.Amberdata]: amberdata,
-  [DataProvider.Cryptocompare]: cryptocompare,
-  [DataProvider.Coinpaprika]: coinpaprika,
-  [DataProvider.Nomics]: nomics,
-  [DataProvider.Coinmarketcap]: coinmarketcap,
-  [DataProvider.Coingecko]: coingecko,
-  [DataProvider.Coinapi]: coinapi,
-  [DataProvider.Kaiko]: kaiko,
+const providers: Record<string, DataProviderConfig> = {
+  [DataProvider.Amberdata]: {
+    batchingSupport: false,
+    defaultDataUrl: 'http://localhost:8080',
```

Comment:
What is this supposed to be the URL to? The adapter?

Reply:
Yep. This is yet to be defined, but should get the `url` from the discovery service